database/contexts/api_v2.c

```diff
@@ -1648,6 +1648,37 @@ static void contexts_v2_alert_transition_callback(struct sql_alert_transition_da
     }
 }
 
+static void contexts_v2_alert_eval_to_json(BUFFER *wb, struct rrdcontext_to_json_v2_data *ctl, bool debug) {
+    struct health_virtual hv = {
+        .debug = debug,
+        .chart = ctl->request->alert_eval.chart,
+        .context = ctl->request->alert_eval.context,
+        .calc = ctl->request->alert_eval.calc,
+        .lookup = ctl->request->alert_eval.lookup,
+        .warn = ctl->request->alert_eval.warn,
+        .crit = ctl->request->alert_eval.crit,
+        .after = ctl->window.after,
+        .before = ctl->window.before,
+    };
+
+    int min_run_every = (int)config_get_number(CONFIG_SECTION_HEALTH, "run at least every seconds", 10);
+    if(min_run_every < 1) min_run_every = 1;
+
+    buffer_json_member_add_array(wb, "alert_eval"                     );
```

Comment:
Hello Manolis,

Is there a special reason to have this space after `alert_eval`?